examples/phpinfo.php

```diff
@@ -1,2 +1,3 @@
 <?php
-phpinfo();
\ No newline at end of file
+phpinfo();
+?>
```

Comment:
Closing tag are not recommended at all, it can leads to "ghost line break" or header/buffer problem.
That's ok for this sample, but don't keep this habit ;)